### PR TITLE
New python packages: py-pymetis, py-triangle, py-meshpy

### PR DESCRIPTION
**Pull request by samcom12**
Added new python packages - py-metis, py-triangle, and py-meshpy

### CODE REVIEW

**Comment by samcom12** [comment]
@alalazo Could you please review and merge so that I can create new PR for the dependant package.

**Comment by tldahlgren** [comment]
Note there is overlap between this PR and #24269

File: var/spack/repos/builtin/packages/py-meshpy/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-meshpy/package.py]:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/py-pymetis/package.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pymetis/package.py:1]:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]

**Comment by adamjstewart** [comment]
@samcom12 can you submit a separate PR for each package.py file you add/modify? This makes it easier for me to review and prevents overlap between PRs.

**Comment by samcom12** [comment]
Okay ... will use separate PR for each package.

**Pull request closed without merging**